### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by fwgbbbbwth** [GaijinEntertainment/fmod_studio_warthunder_for_modders#98]
Sir, I want to know how to use this repository. I can't use the sound module after the war thunder update. The same is true when I downloaded it again today. I hope you can help. Thank you

**Comment by fwgbbbbwth** [comment]
Incidentally, the problem of my voice module appears in the voice packet and RWR system. And without mod, my RWR has no sound

**Comment by Uchuujin01** [comment]
Hello! Do you mean you can't use someone's sound mod? Or is it the sound mod made by you doesn't work anymore?

**Comment by OXEPU3M** [comment]
> > 你好！ 你是说你不能使用某人的声音模组吗？ 还是你制作的音响模组不再起作用了？
> 
> Hello, sir, I can't use the sound modules made by all players at present. I have correctly added codes and folders, but they don't work. Do I need to install other files

Hello.
Do you use steam version? There can be a bug to install game twice. Check your files.
If you really did everything right this is the only reason.

**Comment by fwgbbbbwth** [comment]
> > > 你好！ 你是说你不能使用某人的声音模组吗？ 还是你制作的音响模组不再起作用了？
> > 
> > 
> > 你好，先生，我不能使用目前所有玩家的声音模块。我已经正确地添加了代码和文件夹，但它们不起作用。我是否需要安装其他文件
> 
> 你好。 您是否使用蒸汽版本？安装游戏两次可能会有错误。检查您的文件。 如果你真的做的每件事都是正确的，这是唯一的原因。

Yes, I use the steam version. I have checked all the files and even downloaded the game again. Unfortunately, it still can't be used
